be/src/exprs/runtime_filter.h

```diff
@@ -594,9 +614,25 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
 
     void insert_null() { _has_null = true; }
 
-    CppType min_value() const { return _min; }
+    CppType min_value(ObjectPool* pool) const {
+        if constexpr (IsSlice<CppType>) {
+            std::lock_guard<std::mutex> lk(_slice_mutex);
```

Comment:
why we have `slice_mutex`? to protect from race condiction of pool allocation?

if that, maybe you can name it as `pool_mutex`?

Reply:
This is used to protect _slice_min/_slice_max. _min point to the std::string of _slice_min, so lock it first, copy the string, and then release it. Otherwise, when get, other threads will replace the std::string in _slice_min. ,